save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/info/UserInfo.kt

```diff
@@ -19,10 +21,17 @@ import kotlinx.serialization.Serializable
  * @property twitter
  * @property organizations
  * @property globalRole
+ * @property id
+ * @property isActive
+ * @property oldName
+ * @property originalLogins
  */
 @Serializable
 data class UserInfo(
     val name: String,
+    val id: Long? = null,
+    val oldName: String? = null,
+    val originalLogins: List<String?> = emptyList(),
```

Comment:
Should be list of pairs, together with source?

Reply:
or even a map

Reply:
No. OriginalLogins contains both the name and the source

Reply:
In `UserInfo` you store only names of these logins

Reply:
you concatenated it?

Reply:
No. These are two different fields.